Source/Atomic/Network/Network.cpp

```diff
@@ -596,6 +654,49 @@ void Network::ConfigureNetworkSimulator()
         i->second_->ConfigureNetworkSimulator(simulatedLatency_, simulatedPacketLoss_);
 }
 
+void Network::ClientConnectToMaster(const String& address, unsigned short port)
+{
+    masterServerClient_.ConnectToMaster(address, port);

```

Comment:
Good point. So maybe create a Matchmaking system and move all the MasterClient stuff into that? Call it AtomicMatchMaking?